fjmurray-Wayne.md

```diff
@@ -0,0 +1,10 @@
+# Wayne, PA
+
+### Restaurants
+
+### Local Attractions
+
+- Philadelphia
```

Comment:
@fjmurray I've only been to the airport ✈️  -- but I hear that Philly has a lot of history. Do you have any suggestions?